packages/neondb/src/lib/seed-database.ts

```diff
@@ -0,0 +1,14 @@
+import { neon } from "@neondatabase/serverless";
+import { getSqlCommands } from "./utils/fs.js";
+
+export async function seedDatabase(
+	schemaFilePath: string,
+	connectionString: string,
+) {
+	const client = neon(connectionString);
+	const commands = getSqlCommands(schemaFilePath);
+
+	for (const command of commands) {
+		await client.query(command);
```

Comment:
[curious/not sure] Have you tried just passing the whole file? Do we really need to spilt the commands?

Reply:
🙏 thanks for the context! Yeah your solution is perfectly fine.